Update readme (#5)
LAzyDev-101/Dev-mountain-hackthon-contract: README.md

# **Dev. Mountain Tech Festival(Thailand) 2022 - Solidity Contract**

## What is Dev. Mountain Tech Festival  

> Dev. Mountain Tech Festival is Thailand tech events that setup at

Reviewer: Dev. Mountain Tech Festival เป็นสิ่งเดียว : events -> event
ปล. ถ้าผมเข้าใจพลาดขออภัย

> 18-20 March 2022 Toscana Valley Khao Yai.This event is have many top-tier Thai developers as speaker and have many different topic for developers or anyone who interested in technology

Reviewer: - This event is have ... -> This event has ... ตรงนี้ event เป็นเอกพจน์ (ไม่เติม s) คิดว่าใช้ has ไปเลย แล้วตอนนี้เราเล่า fact ปกติใช้ present simple ได้ เลยคิดว่าใช้แค่ has เป็น verb แท้ตัวเดียวพอแล้ว
- and have many different ... -> and has many different เพราะประธานคือ this event เหมือนกับประโยคแรก
- many different topic ... -> many different topics เพราะมีหลาย topic
ปล. ถ้าผมเข้าใจพลาดขออภัย

> Other way this tech events is have the blockchain hackathon inside it 

Reviewer: in additional, this tech event also has blockchain hackathon inside it

Author: Fixed here -> https://github.com/LAzyDev-101/Dev-mountain-hackthon-contract/pull/5/commits/a48f2fd7df405b91e50f76963fa8b52e6e9e3465

> this Hackathon is having 5 team that passing the first round before can participate in the main event .

Reviewer: ตรงนี้ขอเน้นช่วงแข่งออนไลน์และเน้นย้ำว่ามีทีมแค่ 5 ทีมผ่านเข้ามาเพิ่มหน่อย เพื่อความเท่ แล้วก็ขออธิบายเป้าหมายของงานเพิ่มนิดนึง เพราะข้างบนเรายังไม่ได้พูดถึงว่า hack นี้ทำอะไร นอกจากบอกว่าเป็น blockchain hack, ทำทำไม และ impact อยู่ที่ไหน
In this hackathon, participants have to pass an online smart contract competition. Finally, there were 5 teams participating in the main event and facing 5 unique domains, which come from different businesses. The owner of each problem also entered and waited to hear how competitors use blockchain to solve their problems effectively in this event.

[Facebook page](https://www.facebook.com/devmountaintechfestival/?ref=page_internal)
## About this repo
This repository is for the Solidity Contract (ERC721) <br>
:bookmark_tabs: -> [Contract address](https://testnet.bscscan.com/address/0x4b5e5d9fe104722e75b9b671a20efa31c9d2bd75#code) <br>
:newspaper: ->        [Ours Presentation (Thai)](https://docs.google.com/presentation/d/16d5bD5Ru74sfditDaeJPu73X5vpyMydQS6dL8hUu3EQ/edit#slide=id.g11e4d075bea_1_27)

> ### Ours Hackathon problem that we got

Reviewer: Ours Hackathon problem that we got -> The hackathon problem that we got 
ตรงนี้มี that we got บอกความเป็นของเราแล้ว ปล่อยให้ hackathon problem เป็นคำนามเฉยๆดีกว่า our กับ we ความหมายมันดูซ้อนกัน
ปล. ถ้าผมเข้าใจพลาดขออภัย

**Certificate verification for education** <br> 

> The main goal is want to removes suspicion about the certificatecontent and the issuer educational institution.

Reviewer: - ตรง certificatecontent มันเป็นคำเดียวกันเลยหรือเป็น certificate content อันนี้สงสัย พอดี ไม่คุ้นคำนี้
- The main goal is want to removes ... ตรงนี้เราใช้ to เพื่ออธิบายจุดประสงค์ปกติจะใช้ to-infinitive กันคิดว่าไม่ต้องเติม s ที่ remove ใช้เป็น The main goal is want to remove ... ไปเลย

> And Yeah!! we win this hackathon 
<br>

![image](https://user-images.githubusercontent.com/30498964/168614013-f3be9e2c-c360-4fbd-9f85-6d24a635f407.png)

> ### Ours solutions

Reviewer: Ours solutions -> Our solutions ปกติ Ours จะใช้ตอนเป็นกรรมแล้ว ref มาหา noun ข้างหน้า

> We creating platform named **Eduprove** (Certificate verification online platform)

Reviewer 2: Should be something like this? "We created a platform name **Eduprove** (Certificate verification online platform)"

> We creating platform named **Eduprove** (Certificate verification online platform)

Reviewer 3: ```suggestion
We created a platform named **Eduprove** (Certificate verification online platform)
```

By hashing certificate data issued from whitelisted educational institutions and storing it on the blockchain.

> ## Our team member

Reviewer: member -> members

> That storing hashing of certificate data on blockchain and using some whitelist flow with centralized organization that issuer educational institution

Reviewer 3: ```suggestion
By hashing certificate data issued from centralized organization's whitelisted educational institutions and storing it on the blockchain.
```

> That storing hashing of certificate data on blockchain and using some whitelist flow with centralized organization that issuer educational institution

Reviewer 3: ```suggestion
By hashing certificate data issued from whitelisted educational institutions and storing it on the blockchain.
```
I prefer this one.
